src/racer/util.rs

```diff
@@ -23,6 +27,114 @@ pub fn is_ident_char(c: char) -> bool {
     c.is_alphanumeric() || (c == '_') || (c == '!')
 }
 
+// to ignore C-Style Comment
+#[derive(Debug)]
+enum CommentAutomatonState {
+    None,
+    Start, // just / occurs
+    InComment, // /* ...
+    BeforeEnd, // /* ... *
+    InSlash, // like ///// (maybe not useless)
```

Comment:
Please use rustdoc comments for each enum variant, rather than postfixed comments.